interbotix_ros_xsarms/examples/interbotix_xsarm_gravity_compensation/launch/interbotix_gravity_compensation.launch.py

```diff
@@ -0,0 +1,127 @@
+
+from typing import List
+
+from launch import LaunchDescription
+from launch.actions import (
+    IncludeLaunchDescription,
+    DeclareLaunchArgument,
+    OpaqueFunction
+)
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import (
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+from interbotix_xs_modules.xs_common import get_interbotix_xsarm_models
+
+
+def launch_setup(context, *args, **kwargs):
+    # Define the launch arguments
+    robot_model_launch_arg = LaunchConfiguration('robot_model')
+    robot_name_launch_arg = LaunchConfiguration('robot_name')
+    motor_specs_launch_arg = LaunchConfiguration('motor_specs')
+
+    # Create the gravity compensation node
+    gravity_compensation_node = Node(
+        package='interbotix_gravity_compensation',
+        executable='interbotix_gravity_compensation',
+        name='gravity_compensation',
+        namespace=robot_name_launch_arg,
+        output='screen',
+        emulate_tty=True,
+        parameters=[{'motor_specs': motor_specs_launch_arg}]
+    )
+
+    # include the xsarm_control_launch with arguments
+    xsarm_control_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource([
+            PathJoinSubstitution([
+                FindPackageShare('interbotix_xsarm_control'),
+                'launch',
+                'xsarm_control.launch.py'
+            ])
+        ]),
+        launch_arguments={
+            'robot_model': robot_model_launch_arg,
+            'robot_name': robot_name_launch_arg,
+        }.items()
+    )
+
+    return [gravity_compensation_node, xsarm_control_launch]
+
+
+def generate_launch_description():
+    # Create a list of all the launch arguments
+    declared_arguments: List[DeclareLaunchArgument] = []
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            'robot_model',
+            choices=('wx250s', 'aloha_wx250s'),
+            description='model type of the Interbotix Arm such as `wx200` or `rx150`.'
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            'robot_name',
+            default_value=LaunchConfiguration('robot_model'),
+            description=(
+                'name of the robot (typically equal to `robot_model`, but could be anything).'
+            ),
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            'motor_specs',
+            default_value=[PathJoinSubstitution([
+                FindPackageShare('interbotix_xsarm_gravity_compensation'),
+                'config',
+                "motor_specs_"]), LaunchConfiguration('robot_model'), '.yaml'
+            ],
+            description="the file path to the 'motor specs' YAML file.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            'mode_configs',
```

Comment:
This launch arg isn't being passed down to the xsarm_control launch include.

Reply:
Since the mode_configs is no longer used, this argument is removed in the following commit.